### ISSUE TITLE
RFE: Display "storage cluster" resource as a required custom resource for OCS on UI 

### ISSUE THREAD

**Issue by afreen23** [openshift/ocs-operator#756]
**Proposal:**

Openshift console uses `operatorframework.io/initialization-resource` annotation to display a custom resource as required on UI. OCS can leverage this annotation to allow displaying "storage cluster" CR as a required resource.

This will make users to know what to start with after installing the OCS operator from UI, thereby providing an improved UX

**UI mocks after adding the annotation in operator CSV**

http://openshift.github.io/openshift-origin-design/designs/administrator/olm/operand-creation-at-operator-install/

**Annotation Usage Example (Portworx):**
```
{
  "metadata": {
    "annotations": {
      "operatorframework.io/initialization-resource": "{\n    \"apiVersion\": \"core.libopenstorage.org/v1alpha1\",\n    \"kind\": \"StorageCluster\",\n    \"metadata\": {\n      \"name\": \"portworx-initvalue\",\n      \"namespace\": \"test-operator\"\n    },\n    \"spec\": {\n      \"userInterface\": {\n        \"enabled\": true\n      },\n      \"autopilot\": {\n        \"enabled\": false\n      }\n    }\n  }\n"
    }
  }
}
```
_Annotation key:_ `operatorframework.io/initialization-resource` 
_Annotation value_:  a single value of the default yaml of the CR which should have - apiVersion, kind and metadata fields. Other fields can be added as well as per the CRD

cc @obnoxxx

**Comment by jarrpa** [comment]
Is there a Jira story about this? Since this is focused on the product experience, it should be tracked in a place where PM and Engineering management have visibility to it so we can plan for it accordingly.

**Comment by afreen23** [comment]
Yes, it is included in 4.7 planning under "UX research based enhancments [Feature]". 
Surely after planning exit we will have a JIRA created.

**Comment by bipuladh** [comment]
https://github.com/openshift/ocs-operator/pull/907 
Fixes this issue
